lib/digsync_web/live/groups_live.ex

```diff
@@ -39,15 +44,12 @@ defmodule DigsyncWeb.GroupsLive do
 
   defp fetch_groups() do
     Group
-      |> Accounts.read!()
-      |> sanitize_groups()
+    |> Accounts.read!()
```

Comment:
So just as a side note, 

Accounts.read!() returns the group or hard errors, in practice we don't want to use that. For now its fine, but remember you cannot pattern match on {:ok, group} as fetch_groups() returns the group or nil and not in a tuple.

Reply:
Most bang methods in elixir do that, they return the thing or they return a hard error. So usually bang methods are best in iex or in seeding because we want the app to hard fail (also in GenServers but we havn't talked about those yet)